x-pack/test/security_solution_cypress/cypress/e2e/explore/navigation/navigation.cy.ts

```diff
@@ -352,10 +351,6 @@ describe('Serverless side navigation links', { tags: '@serverless' }, () => {
     navigateFromHeaderTo(ServerlessHeaders.ENDPOINTS, true);
     cy.url().should('include', ENDPOINTS_URL);
   });
-  it('navigates to the Fleet page', () => {
-    navigateFromHeaderTo(ServerlessHeaders.FLEET, true);
-    cy.url().should('include', FLEET_URL);
-  });
```

Comment:
Per @paulewing, the removal of Fleet from the primary nav "was purposeful to focus those asset mngmt workflows in a single asset UX"